<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Bg</class>
 <widget class="QDialog" name="Dialog_Bg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Background</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="set_bg">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reset_bg">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_bg">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelr">
           <property name="text">
            <string>R:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="bg_col_r">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelb">
           <property name="text">
            <string>B:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="bg_col_b">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="labelgg">
           <property name="text">
            <string>G:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="bg_col_g">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QGraphicsView" name="graphicsView">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
